dvc/command/remove.py

```diff
@@ -13,6 +14,9 @@ def run(self):
         for target in self.args.targets:
             try:
                 self.repo.remove(target, outs=self.args.outs)
+            except StageNotFound:
+                logger.error(f"{target} is not under DVC control")
```

Comment:
We don't support removing an output, only the `.dvc` file or the stage completely. If we want to support it, we need to make the `repo.remove` API support it.

Reply:
Let me comment on the issue itself.